permissions/plugin-zohosprints.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "zohosprints-plugin"
```

Comment:
Wrong. This should be `artifactId`.

Reply:
@daniel-beck  zohosprints is an artifactId.

Reply:
```suggestion
name: "zohosprints"
```

Reply:
@daniel-beck I'll do necessary changes